lib/src/components/NxSmallVulnerabilityCounter/__tests__/NxSmallVulnerabilityCounter.test.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { rtlRender, rtlRenderElement, runTimers } from '../../../__testutils__/rtlUtils';
+
+import NxSmallVulnerabilityCounter, { Props } from '../NxSmallVulnerabilityCounter';
+
+describe('NxSmallVulnerabilityCounter', function() {
+  const quickRender = rtlRender<Props>(NxSmallVulnerabilityCounter, {}),
+      renderEl = rtlRenderElement<Props>(NxSmallVulnerabilityCounter, {});
+
+  it('renders nothing if all five indicators are undefined', function() {
+    expect(renderEl()).not.toBeInTheDocument();
+  });
+
+  it('renders nothing if all five indicators are null', function() {
+    const el = renderEl({
+      criticalCount: null,
+      highCount: null,
+      mediumCount: null,
+      lowCount: null,
+      noneCount: null
+    });
+    expect(el).not.toBeInTheDocument();
+  });
+
+  it('adds specified classNames to the element in addition to the defaults', function() {
+    const el = renderEl({ criticalCount: 0, className: 'test-class' }),
+        defaultEl = renderEl({ criticalCount: 0 })!;
+
+    expect(el).toHaveClass('test-class');
+
+    for (const cls of Array.from(defaultEl.classList)) {
+      expect(el).toHaveClass(cls);
+    }
+  });
+
+  it('adds specified attrs', function() {
+    const el = renderEl({ criticalCount: 0, id: 'test-id', lang: 'en' });
+
+    expect(el).toHaveAttribute('id', 'test-id');
+    expect(el).toHaveAttribute('lang', 'en');
+  });
+
+  it('renders a counter with an accessible name of Critical when criticalCount is specified', async function() {
+    const component = quickRender({criticalCount: 0});
+
+    await runTimers();
+
+    const counter = component.getByTitle('Vulnerability: Critical (9.0\u201410.0)');
```

Comment:
Replace the escape with the literal `—` throughout.

Reply:
Replaced here: [57fd872](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/57fd87207c758ad46cac4932ef0ba740463ae6b9)

Reply:
> Replaced here: [57fd872](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/57fd87207c758ad46cac4932ef0ba740463ae6b9)

There's more of them; search the file.

Reply:
I've replaced them all here: [219258d](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/219258d1462a4055a6ad79bc6f1ee90d29250797)